insights/specs/insights_archive.py

```diff
@@ -239,6 +239,7 @@ class InsightsArchiveSpecs(Specs):
     saphostctl_getcimobject_sapinstance = simple_file("insights_commands/usr.sap.hostctrl.exe.saphostctrl_-function_GetCIMObject_-enuminstances_SAPInstance")
     saphostexec_status = simple_file("insights_commands/usr.sap.hostctrl.exe.saphostexec_-status")
     saphostexec_version = simple_file("insights_commands/usr.sap.hostctrl.exe.saphostexec_-version")
+    satellite_enabled_features = simple_file("insights_commands/curl_-sk_https_..localhost_9090.features")
```

Comment:
For `curl` command, we can only collect ut by using the `tools.cat` of insights at this point:

```"insights_commands/python_-m_insights.tools.cat_--no-header_ satellite_enabled_features"```